### PR TITLE
Fix Alignment #2115

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR for issues: #2115 

## How Has This Been Tested?
Manually Tested 

## Screenshots (jpeg or gifs if applicable):
image

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: assets/scss/admin/forms.scss [hunk 1]
```diff
@@ -508,16 +508,6 @@ ASIDE
 				cursor: pointer;
 			}
 
-			.give-remove:hover {
```
Comment (DevinWalker) [inline assets/scss/admin/forms.scss]: @raftaar1191 don't remove the hover effect. Just fix the hover so it's no jumpy.

Comment (raftaar1191) [reply assets/scss/admin/forms.scss]: Oh I thought I have to remove that 

Okay fixing it again @DevinWalker

**Comment by raftaar1191** [comment]
GIF link: http://recordit.co/8akahGsCSl

This PR is ready for review, please review it  @DevinWalker

**Comment by raftaar1191** [comment]
Fixed circle SIZE

http://recordit.co/3fBcN1btdX

**Comment by DevinWalker** [comment]
Looks good 👍

**Pull request merged**